lang/en/tool_userupsert.php

```diff
@@ -26,3 +26,14 @@
 defined('MOODLE_INTERNAL') || die();
 
 $string['pluginname'] = 'User upsert';
+$string['privacy:metadata'] = 'The User upsert plugin does not store any personal data.';
+$string['fields'] = 'Web service fields';
+$string['fields_desc'] = 'A list of expected user fields for the web service and their descriptions. The field name followed by a pipe (|) and the description of that field. Only one field per line, separated by a line break in between each field.
+Those fields will appear in the <a href="/admin/webservice/documentation.php">Moodle API Documentation documentation</a> for tool_userupsert_upsert_users web service.
+
+For example:
+<pre>
+userName | Username field.
+authType | Auth type of the user.
```

Comment:
Are the full stops expected in this example? I don't know how the description will be displayed later.

Reply:
![image](https://user-images.githubusercontent.com/4626431/136291037-52a1c3af-d5fc-494b-9bea-08141f35e91c.png)

Reply:
Description of the fields, not the description of the admin settings